D1673/P1673.md

```diff
@@ -6210,22 +6226,26 @@ Instead, it interprets such indices
 as if the matrix were symmetric. <i>-- end note]</i>
 
 ```c++
-template<class OtherExtents>
-  friend constexpr bool
-    operator==(const mapping&, const mapping<OtherExtents>&) noexcept;
+constexpr index_type stride(rank_type r) const noexcept;
 ```
 
-[14]{.pnum} *Constraints:* `OtherExtents::rank()` equals `rank()`.
+[15]{.pnum} *Preconditions:*
+
+   * [15.1]{.pnum} `is_strided()` is `true`, and
 
-[15]{.pnum} *Returns:* `true` if and only if
-    for 0 ≤ `r` < `rank()`,
-    `m.extent(r)` equals `extent(r)`.
+   * [15.1]{.pnum} `r < extents_type::rank()` is `true`.
+
+[16]{.pnum} *Returns:* 1 if `extent(0)` is less than 2, else 0.
```

Comment:
The `0` case should only kick in if `is_strided()` is `false`, which is a precondition violation anyway.  I like that we're defining the return value regardless, but LWG may ask about this.

Reply:
oh ok. Lets take it out. is_strided() is clearly a precondiiton.